cmd/state-tray/internal/menu/projects.go

```diff
@@ -16,9 +15,11 @@ type LocalProjectsUpdater struct {
 }
 
 type localProjectsMenuItem struct {
-	menuItem *systray.MenuItem
-	project  *graph.Project
-	cancel   context.CancelFunc
+	menuItem  *systray.MenuItem
+	namespace string
+	location  string
+	done      chan struct{}
```

Comment:
Nit: name this `close`. `done` conveys the menuitem itself is doing something that it needs to finish.